README.md

```diff
@@ -255,6 +255,46 @@ irods_http_api /path/to/config.json
 
 To stop the server, you can use **CTRL-C** or send **SIGINT** or **SIGTERM** to the process.
 
+### OpenID Connect
+~~To run the server with OpenID Connect, you must perform the following actions:~~
+
+Some Additional configuration is required to run the OpenID Connect portion of the HTTP API.
+Following are a few points of interest.
```

Comment:
might be able to condense these three lines into one.

Reply:
Yes, 259 may be removed. Left there if we prefer that over 261-262.

Reply:
In two lines now, though can be one if desired. Might just need a little word shuffle.